gcp/appengine/frontend3/src/templates/vulnerability.html

```diff
@@ -68,6 +68,26 @@ <h2 class="package-header">
           strip_scheme }}</span>
       </h2>
       <div class="mdc-layout-grid">
+        <div class="vulnerability-package-subsection mdc-layout-grid__inner">
+          <h3 class="mdc-layout-grid__cell--span-3">
+            Source Details
+          </h3>
+          <div class="mdc-layout-grid__cell--span-9">
+            <dl>
+              {%- if affected['package']['name'] -%}
+              <dt>Package Name</dt>
+              {# TODO(andrewpollock): link out to the package in the ecosystem -#}
+              <dd>{{ affected['package']['name'] }}</dd>
+              {%- endif -%}
+              {%- if vulnerability.repo -%}
+              <dt>Repository</dt>
+              {# TODO(andrewpollock): it's permissible to have more than one repo, so handle this -#}
+              <dd><a href="{{ vulnerability.repo }}" target="_blank" rel="noopener noreferrer">{{ vulnerability.repo
+                  }}</a></dd>
```

Comment:
nit: the newline before this seems a bit weird. is it required?

Reply:
VS Code's being overly... something about formatting on save.